packages/auth/src/server.ts

```diff
@@ -104,7 +105,7 @@ export const {
       };
     },
   },
-  adapter: PrismaAdapter(prisma),
+  adapter: PrismaAdapter(prisma) as Adapter,
```

Comment:
feels weird we have to cast for official adapters. is this from the docs?

Reply:
There is a weird type mismatch error that occurs without an explicit cast, it seems there are some incompatibilities between `next-auth` and `@auth/prisma-adapter` Adapter types.
Link: https://github.com/nextauthjs/next-auth/issues/7727

Reply:
this issue seems to suggest the type error is supposed to be resolved in 5. oh well i’ll look deeper later